apps/api/src/mail/emails/workspace-removal.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  Body,
+  Container,
+  Head,
+  Heading,
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text
+} from '@react-email/components'
+import * as React from 'react'
+interface WorkspaceRemovalEmailProps {
+  workspaceName: string
+  removedOn: string
+}
+
+export const WorkspaceRemovalEmail = ({
```

Comment:
Rename this to `RemovedFromWorkspaceEmail`

Reply:
Done